### ISSUE TITLE
When evaluate on the sintel train datasets,the epe result is nan

### ISSUE THREAD

**Issue by ljp1997** [princeton-vl/RAFT#98]
Hi,I download the pre-training models:raft-chairs.pth,raft-things.pth,raft-sintle.pth,and I use evaluate.py.
I evaluate on the validation set of flyingchairs dataset using the raft-chairs.pth and the epe result is fine.
But when I evaluate on the sintel train dataset using the raft-chairs.pth,the epe results become nan,then 
I use the raft-things.pth,raft-sintel.pth,the epe results are nan,too.Have you encountered this situation?
How can I solve this problem？Thank you very much.

**Comment by ljp1997** [comment]
At first I made an inference on the old graphics card, which led to the wrong result, then I made an inference on the 1070, and the result was good.
